org.jacoco.doc/docroot/doc/changes.html

```diff
@@ -38,6 +38,14 @@ <h3>Fixed Bugs</h3>
       (GitHub <a href="https://github.com/jacoco/jacoco/issues/672">#672</a>).</li>
 </ul>
 
+<h3>API Changes</h3>
+<ul>
+  <li>The XML report now has an optional attribute <code>sourcefilename</code>
```

Comment:
What do you think about addition of note that for better forward compatibility we recommend consumers of XML report to not fail (ignore / warn) on unknown attributes / nodes ?

Reply:
Not sure whether our change log is the right place to advocate robust interface design line *tolerant reader*.

Reply:
Well, you right - not really a correct place.